generators/client/templates/angularjs/src/main/webapp/app/services/auth/_sessions.service.js

```diff
@@ -26,7 +26,7 @@
     Sessions.$inject = ['$resource'];
 
     function Sessions ($resource) {
-        return $resource('<%- apiUaaUrlPrefix %>api/account/sessions/:series', {}, {
+        return $resource('api/account/sessions/:series', {}, {
```

Comment:
Just for my understnading why is the uaa url or server api url not needed here?

Reply:
This file is not copied for UAA, only session-auth